### PR TITLE
Feat::start working on new features

### PR DESCRIPTION
**Pull request by Atharv1848**
please look into changes and verify i added in readme.md

### CODE REVIEW

**Comment by rajiv-shrivastava** [comment]
Hey atharva please add better message

File: README.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
  - Button ,Click etc event handling with Jest
  - How to use Enzyme with jest
 
+ -React Error Boundaries
```
Comment (rajiv-shrivastava) [inline README.md]: Leave a space

**Review by rajiv-shrivastava — COMMENTED** [review]
Leave proper message

**Comment by Atharv1848** [comment]
> Hey atharva please add better message

ok sure

File: README.md (hunk 1 above)
Comment (Atharv1848) [reply README.md]: done

**Pull request merged**